src/core/packages/elasticsearch/server-internal/src/elasticsearch_service.ts

```diff
@@ -191,7 +192,7 @@ export class ElasticsearchService
     }
 
     return {
-      client: this.client!,
+      client: omit(this.client!, ['close']),
```

Comment:
I noticed that the client is exposed with the method `closed` even when our types claim that it doesn't